### ISSUE TITLE
Request for training configurations and reproduction scripts for RoboMemArena baselines

### ISSUE THREAD

**Issue by Lin-Jia-Hui24** [OpenHelix-Team/RoboMemArena#18]
Hi, thank you for releasing RoboMemArena and the evaluation code! We are currently trying to reproduce the baseline results reported in the paper and use RoboMemArena for our own VLA experiments.

We noticed that the repository already provides the dataset, the 1–26 task evaluation pipeline, reference VLM/VLA evaluation code, and the PrediMem predictive-coding add-on. However, we could not find the exact training configurations used to obtain the reported results for the different baseline models.

In particular, we are interested in the configurations used for the models evaluated in RoboMemArena, including:

* π0.5
* MemoryVLA
* MemER
* HiF-VLA
* PrediMem

Could you please provide the training configurations used for these experiments?

The information that would be especially helpful includes:

1. **Training data**

   * Which part of the released RoboMemArena dataset was used for each model?
   * Was training performed on `subtask_data`, `full_trajectory`, or both?
   * Were all 26 tasks used jointly for training?
   * Were there any additional datasets mixed with RoboMemArena data?

2. **Training schedule**

   * Number of epochs or total optimization steps
   * Global/per-device batch size
   * Learning rate and LR scheduler
   * Optimizer
   * Warmup steps/ratio
   * Gradient accumulation
   * Number/type of GPUs used, if possible

3. **Model-specific configuration**

   * Exact pretrained checkpoint used as initialization
   * Which modules were frozen or fine-tuned
   * Action chunk size / prediction horizon
   * Observation history length and camera inputs
   * Image resolution / preprocessing
   * Any model-specific memory hyperparameters used for MemoryVLA, MemER, HiF-VLA, and PrediMem

4. **Evaluation configuration**

   * Exact checkpoint selection criterion
   * Number of evaluation trials/seeds per task
   * Replanning/action execution interval
   * Any model-specific evaluation settings
   * Dataset/evaluator version or commit corresponding to the numbers reported in the paper

More importantly, **would it be possible to release the training and evaluation scripts/config files used to reproduce the reported baseline results?**

For example, having configs/scripts corresponding to something like

```text
configs/
  pi05_robomemarena.yaml
  memoryvla_robomemarena.yaml
  memer_robomemarena.yaml
  hifvla_robomemarena.yaml
  predimem_robomemarena.yaml

scripts/
  train_*.sh
  eval_*.sh
```

would make it much easier for the community to reproduce the benchmark numbers and compare new memory mechanisms under exactly the same training protocol.

Even if releasing the complete training code is inconvenient, sharing the **exact commands/configurations used for the reported experiments** would already be extremely helpful.

Thank you very much for your work and for making RoboMemArena publicly available!
